lib/layout/templated.dart

```diff
@@ -63,6 +63,22 @@ mixin TemplatedWidgetState<W extends StatefulWidget> on State<W> {
     }
     return widgets;
   }
+  // ------------------------ Render the listview children ----------------------------------
+  DataScopeWidget? buildWidgetsForListView(BuildContext context, List dataList, ItemTemplate itemTemplate,int itemIndex) {
+    DataScopeWidget? widgets;
+    ScopeManager? parentScope = DataScopeWidget.getScope(context);
```

Comment:
Try not to copy and paste code. Can you create a function for creating individual widget inside templated.dart, and have both places use this block?